2020-global/talks/03_LATAM/02-glowcoil-published.md

```diff
@@ -0,0 +1,45 @@
+**Ochre: Highly portable GPU accelerated vector graphics**
```

Comment:
Should we just remove the plaintext version?

Reply:
We could, I guess.